src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -64,6 +64,8 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
 
     case VenueTemplate.animatemap:
       template = <AnimateMap venue={venue} />;
+      // NOTE: Remove the back button, because we don't need it in AnimateMap
```

Comment:
// @debt type of comment?

Reply:
done in https://github.com/sparkletown/sparkle/pull/2056/commits/381df8177cc2405fd530e515050b13443f88651f